cmd/loadTests.go

```diff
@@ -1229,24 +1299,28 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 					}
 					if deploymentIsDone {
 						dur := lastUpdateTimeOfDone.Time.Sub(creationTimestamp.Time)
+						MetricsWrapper(MetricsController, metricsConstants.CollectorDeployments, metricsConstants.MetricTypeGuage, metricsConstants.MetricDeploymentsCreationTimeGauge, float64(dur))
```

Comment:
```suggestion
						MetricsWrapper(MetricsController, metricsConstants.CollectorDeployments, metricsConstants.MetricTypeGuage, metricsConstants.MetricDeploymentsCreationTimeGauge, dur.Seconds())
```
Instead of nanoseconds, please push seconds.